pkg/apis/experimental/types.go

```diff
@@ -639,3 +639,36 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// Selector is a label query over a set of API objects. MatchLabels and
+// MatchExpressions are ANDed. An empty Selector matches all objects. A
+// null Selector matches no objects.
+type Selector struct {
+	// MatchLabels is a list of labels.
```

Comment:
The purpose is to preserve the ease of use of the current selector for everyone for whom that is sufficient, and to make it straightforward to port clients to the new selector. Yes, it could be subsumed by MatchExpressions, but while such normalization is convenient for computers, it is less convenient for users. For the same reason, we support `=` and `!=` syntax in the string serialized form used in list and watch.